libr/core/project.c

```diff
@@ -394,6 +448,14 @@ R_API bool r_core_project_save(RCore *core, const char *file) {
 	snprintf (buf, sizeof (buf), "%s.d" R_SYS_DIR "xrefs", prj);
 	r_anal_project_save (core->anal, buf);
 
+	Sdb *rop_db = sdb_ns (core->sdb, "rop", false);
+	if (rop_db) {
+		snprintf (buf, sizeof (buf), "%s.d" R_SYS_DIR "rop", prj);
+		sdb_file (rop_db, buf);
+		sdb_sync (rop_db);
+	}
+
```

Comment:
@radare fixed, same as before, diff can't tell because it sees the second line now